src/PresentationalComponents/Notification/Notification.js

```diff
@@ -109,20 +111,29 @@ const Notification = () => {
   return (
     <React.Fragment>
       <PageHeader>
-        <PageHeaderTitle title="Notification Insights" />
+        <Split>
+          <SplitItem isFilled>
+            <PageHeaderTitle
+              title={`My Notifications | ${bundleDisplayTitle}`}
+              style={{ paddingBottom: '8px' }}
+            />
+            <StackItem>
+              This service allows you to opt-in and out of receiving
+              notifications. Your Organization Administrator has configured
+              which notifications you can or can not receive in their
+              <a href={getBaseName(`/settings/notifications/${bundleName}`)}>
```

Comment:
We should use `chrome`'s api similiar to how dashboard is doing it https://github.com/RedHatInsights/insights-dashboard/blob/master/src/SmartComponents/Drift/DriftCard.js#L43 this way we won't force page refresh, but rather make it SPA like.